### PR TITLE
Rolling PR for flight-service-1.3.X

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**
This work will likely supersede the previous `flight-configure` work.

### CODE REVIEW

File: builders/flight-service/package-scripts/flight-service/postinst [hunk 1]
```diff
@@ -26,7 +26,13 @@
...
-# This directory needs to exist for `flight service configure` to work.
-mkdir -p /opt/flight/opt/service/var/lib
+# Install the reboot cron script to get around the flight-plugin-system-cron
+# rpm file conflict. This can be removed and packaged into the RPM once the
+# conflict is resolved.
+# Remember to update the postrm script as well
+cat << DOC > /opt/flight/etc/cron/reboot/start-enabled-services
+/opt/flight/bin/flight service stack start
+DOC
+chmod u+x /opt/flight/etc/cron/reboot/start-enabled-services
```
Comment (benarmston) [inline builders/flight-service/package-scripts/flight-service/postinst:36]: We don't need this.  The packages `flight-plugin-{system,manual}-systemd-service` exist to fix this issue.

File: builders/flight-service/package-scripts/flight-service/postrm [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# along with this project. If not, see:
+#
+#  https://opensource.org/licenses/EPL-2.0
+#
+# For more information on OpenFlight Omnibus Builder, please visit:
+# https://github.com/openflighthpc/openflight-omnibus-builder
+#===============================================================================
+
+# Remove the cron script
+rm /opt/flight/etc/cron/reboot/start-enabled-services
```
Comment (benarmston) [inline builders/flight-service/package-scripts/flight-service/postrm:30]: We don't need this either.

**Review by benarmston — CHANGES_REQUESTED** [review]
There is already a mechanism for starting enabled services.  The cron job needs to be removed.

**Comment by WilliamMcCumstie** [comment]
Removed

**Comment by benarmston** [comment]
LGTM

**Pull request merged**